examples/five_mem_node/main.rs

```diff
@@ -207,6 +211,15 @@ fn on_ready(
         return;
     }
 
+    // Apply the snashot. It's also necessary with same reason as above.
```

Comment:
```suggestion
    // Apply the snapshot. It's also necessary with same reason as above.
```

Also, which reason? That the program will be blocked forever?